src/Symfony/Component/Form/Extension/Core/Type/MoneyType.php

```diff
@@ -80,6 +83,7 @@ public function configureOptions(OptionsResolver $resolver)
             \NumberFormatter::ROUND_CEILING,
         ]);
 
+        $resolver->setAllowedValues('format', ['float', 'integer']);
```

Comment:
In the context of our date types we use the term `input`.

Reply:
Maybe `input_type` ?